spec/controllers/concerns/user_session_logging_concern_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe UserSessionLoggingConcern, type: :controller do
```

Comment:
```suggestion
describe UserSessionLoggingConcern do
```

RSpec [should infer this automatically](https://github.com/nhsuk/manage-vaccinations-in-schools/blob/b576a6fa3509d54e0351e8f78e543d1b97a8369e/spec/spec_helper.rb#L192).